### PR TITLE
fix: use dedicated validation resource for context-path updates

### PR DESCRIPTION
**Pull request by a-cordier**
see https://gravitee.atlassian.net/browse/APIM-1983
see https://github.com/gravitee-io/issues/issues/9093

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-apnngbuoeg.chromatic.com)

### CODE REVIEW

**Comment by gaetanmaisse** [comment]
Can you also add the link to the GH issue and the commit + PR?

File: gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApiResource.java [hunk 1]
```diff
@@ -898,6 +893,26 @@ public Response promoteAPI(@RequestBody @Valid @NotNull final PromotionRequestEn
         return Response.ok(promotionService.promote(GraviteeContext.getExecutionContext(), promotion.getId())).build();
     }
 
+    @POST
```
Comment (phiz71) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApiResource.java]: I would have added `apiId` in `VeryApiParam` class and used the `verify` resource of `ApisResource` class. WDYT ?

**Comment by gaetanmaisse** [comment]
Same feedback as @phiz71 it seems to me a bit overkill to have a new dedicated endpoint for that

File: gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApiResource.java (hunk 1 above)
Comment (a-cordier) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApiResource.java]: ok done

**Review by phiz71 — APPROVED** [review]

**Pull request merged**